backend/scripts/request_runner.py

```
@@ -0,0 +1,23 @@
+def request_runner(query: str):
+    import requests
+    import json
+
+    response = requests.post(
+        f"http://localhost:8000/chat",
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)